### PR TITLE
Updated disable format

### LINKED ISSUE #2714: Update disable format
We would like to update the disabled format of in all playlist.xml files, here is an example:
from
```
<disabled>
		<comment>https://github.com/adoptium/aqa-tests/issues/1994#issuecomment-705633262</comment>
</disabled>
```
to
```
<disables>
		<disable>
				<comment>https://github.com/adoptium/aqa-tests/issues/1994#issuecomment-705633262</comment>
		</disable>
</disables>
```
similar to the change in OpenJ9 repo: https://github.com/eclipse-openj9/openj9/pull/13105

### PR DESCRIPTION
**Pull request by ManasviGoyal**
fixed #2714 

updated
```
<disabled>
		<comment>https://github.com/adoptium/aqa-tests/issues/1994#issuecomment-705633262</comment>
</disabled>
```

to

```
<disables>
		<disable>
				<comment>https://github.com/adoptium/aqa-tests/issues/1994#issuecomment-705633262</comment>
		</disable>
</disables>
```

### CODE REVIEW

File: jck/compiler.lang/playlist.xml [hunk 1]
```diff
@@ -56,18 +60,22 @@
...
+			</disable>
+		</disables>
+		<disables>
+			<disable>
+				<comment>Disabled on aix due to RTC: 144784</comment>
+				<platform>ppc64_aix(_mixed)?</platform>
+				<version>11</version>
+				<impl>openj9</impl>
+			</disable>
+		</disables>
```
Comment (llxia) [inline jck/compiler.lang/playlist.xml:78]: `<disable>` should be all under one `<disables>`. Please see https://github.com/eclipse-openj9/openj9/blob/master/test/docs/OpenJ9TestUserGuide.md#exclude-test-against-multiple-criteria for more detail. Please check the rest of PR. Thanks.

**Comment by ManasviGoyal** [comment]
@sophia-guo @smlambert corrected please check.

**Comment by renfeiw** [comment]
run aqa --sdk_resource nightly --build_list openjdk --target _sanity --jdk_version 11 --jdk_impl hotspot --platform x86-64_linux

**Review by llxia — APPROVED** [review]
LGTM

**Review by renfeiw — APPROVED** [review]
LGTM, thanks!

**Comment by smlambert** [comment]
Merging now (before full code freeze lifts) as one of the set of low risk PRs (also as opensource day cutoff for PRs eligibility ends today).

**Pull request merged**